deploy.py

```diff
@@ -353,6 +353,7 @@ def build_example(self):
   def build_opensk(self):
     info("Building OpenSK application")
     self._build_app_or_example(is_example=False)
+    self._check_invariants()
```

Comment:
Almost... Here you compile the firmware then run the tests. Just swap the 2 lines and we're good.

Reply:
Sure, done, thx!